cmd/esc/cli/testdata/env-log.yaml

```diff
@@ -40,7 +40,7 @@ stdout: |+
   Author: test-tester <Test Tester>
   Date:   1970-01-01 04:00:00 +0000 UTC
 
-  revision 3
+  revision 3 (tag: stable)
```

Comment:
Would be nice to have a test case with multiple tags

Reply:
Agree--will do that as a follow-up. It's a bit involved.